test/appium/views/chat_view.py

```diff
@@ -114,6 +114,23 @@ def click(self):
         self.wait_for_element().click()
 
 
+class ProfileMuteContactButton(Button):
+    def __init__(self, driver):
+        super(ProfileMuteContactButton, self).__init__(driver, accessibility_id="Mute-item-button")
+
+    def click(self):
+        self.scroll_to_element()
+        self.wait_for_element().click()
+
+
+class ProfileUnmuteContactButton(Button):
+    def __init__(self, driver):
+        super(ProfileUnmuteContactButton, self).__init__(driver, accessibility_id="Unmute-item-button")
+
+    def click(self):
```

Comment:
not sure if we need to define separate class for re-defining only this method, as `scroll_and_click` method exists

Reply:
thanx @churik, refactored